src/llm/http_llm_calculator.cc

```diff
@@ -195,88 +193,81 @@ class OpenAIChatCompletionsRequest {
         it = this->doc.FindMember("repetition_penalty");
         if (it != this->doc.MemberEnd()) {
             if (!it->value.IsDouble())
-                return false;
+                return absl::InvalidArgumentError("repetition_penalty is not a double");
```

Comment:
I think we should go with a "floating point number" instead of a "double" since it's a more general term, better suiting for the API I believe.